app/src/components/authentication/ProtectedRoute.tsx

```diff
@@ -15,7 +15,7 @@ export const ProtectedRoute = ({children}: {children: JSX.Element}) => {
   // redirect to login page if user is not authenticated
   // save location from which user was redirected to login page
   if (!user) {
-    return <Navigate to="/signin" state={{from: location}} replace />;
+    return <Navigate to="/" state={{from: location}} replace />;
```

Comment:
This was done because of issue #552. A feature requested was for the logout to redirect the user to the homepage. Since my merge messed up I was re-adressing it here.